torch/csrc/jit/passes/memory_planning.cpp

```diff
@@ -0,0 +1,159 @@
+
+#include <c10/core/ScalarType.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/memory_planning.h>
+#include <torch/csrc/jit/runtime/static/impl.h>
+
+namespace torch {
+namespace jit {
+
+c10::TensorTypePtr checkIsSupported(const Value& value) {
+  TORCH_CHECK(
```

Comment:
I feel like none of these conditions should trigger an error, unless you are going to catch it somewhere. memory planning is an optimization and as such should bail rather than throw

Reply:
agreed. will change to something else.

Reply:
@makslevental arrgh sorry I meant `allclose` , https://pytorch.org/docs/stable/generated/torch.allclose.html